src/test/java/uk/ac/ebi/ena/webin/cli/WebinCliTest.java

```diff
@@ -543,4 +546,26 @@ public void testPrintManifestHelpReads() {
                 "│Sequence reads in a BAM file. │               │1              │               │\n" +
                 "└──────────────────────────────┴───────────────┴───────────────┴───────────────┘\n");
     }
+
+    @Test
+    public void testInvalidEnaSubmissionToolParameter() throws Throwable {

```

Comment:
- This test is not needed when WebinCliParameters uses an enum instead of a string for the submission tool. Nice idea in itself to collect the submission tool using WebinCliParameters.